modules/logging-release-notes-5-8-1.adoc

```diff
@@ -0,0 +1,126 @@
+
+* Before this update, the Vector set the `default` log level incorrectly. With this update, the correct log level is set by improving the enhancement of regular expression or `regexp` for log level detection. (link:https://issues.redhat.com/browse/LOG-4480[LOG-4480])
+
+* Before this update, the default alias was missing from the initial index in each log output while creating index patterns. As a result, the Kibana users were unable to create index patterns using {es-op}. With this update, adding the missing aliases to {es-op} resolves the issue. As a result, the Kibana users can create index patterns that include the `{app,infra,audit}-000001` indices. (link:https://issues.redhat.com/browse/LOG-4683[LOG-4683])
+
+* Before this update, Fluentd collector pods were in a `CrashLoopBackOff` state due to binding of the Prometheus server on IPv6 clusters. With this update, the collectors work properly on IPv6 clusters. (link:https://issues.redhat.com/browse/LOG-4706[LOG-4706])
+
+* Before this update, the {clo} would undergo numerous reconciliations whenever there was a change in the `ClusterLogForwarder`. With this update, the {clo} disregards the status changes in the collector daemonsets that triggered the reconciliations. (link:https://issues.redhat.com/browse/LOG-4741[LOG-4741])
+
+* Before this update, the Vector log collector pods were stuck in the `CrashLoopBackOff` state on IBM Power machines. With this update, the Vector log collector pods start successfully on IBM Power architecture machines. (link:https://issues.redhat.com/browse/LOG-4768[LOG-4768])
+
+* Before this update, forwarding with a legacy forwarder to an internal Lokistack would produce SSL certificate errors using Fluentd collector pods. With this update, the log collector service account is used by default for authentication and also using the associated token and `ca.crt`. (link:https://issues.redhat.com/browse/LOG-4791[LOG-4791])
+
+* Before this update, forwarding with a legacy forwarder to an internal Lokistack would produce SSL certificate errors using Vector collector pods. With this update, the log collector service account is used by default for authentication and also using the associated token and `ca.crt`. (link:https://issues.redhat.com/browse/LOG-4852[LOG-4852])
+
+* Before this fix, IPv6 addresses would not be parsed correctly after evaluating a host or multiple hosts for placeholders. With this update, IPv6 addresses are correctly parsed. (link:https://issues.redhat.com/browse/LOG-4811[LOG-4811])
+
+* Before this update, it was necessary to create a `ClusterRoleBinding` to collect audit permissions for HTTP receiver inputs. With this update, it is not necessary to create the `ClusterRoleBinding` since the endpoint already depends upon the cluster certificate authority. (link:https://issues.redhat.com/browse/LOG-4815[LOG-4815])
```

Comment:
```suggestion
* Before this update, it was necessary to create a `ClusterRoleBinding` to collect audit permissions for HTTP receiver inputs. With this update, it is not necessary to create the `ClusterRoleBinding` because the endpoint already depends upon the cluster certificate authority. (link:https://issues.redhat.com/browse/LOG-4815[LOG-4815])
```

Reply:
^Per IBM Style, "Use since in relation to the passing of a period of time, not as a synonym for because."